src/main/java/NotificationHRGUI.java

```diff
@@ -24,43 +25,65 @@ public class NotificationGUI extends JFrame implements ActionListener {
+    public NotificationHRGUI(User user) {
         this.frame.setLayout(new GridLayout(1, 2));
         String[] mom = {"12"};
         populateLists(mom);
         createNotificationList(this, unresolvedNotificationPanel, unresolvedNotificationListPanel,
-                unresolvedNotificationLabel, unresolvedNotifications, unresolvedNotificationList,
+                unresolvedNotificationLabel, unresolvedNotificationList,
                 unresolvedNotificationListScroller);
         addButtons(this, unresolvedNotificationPanel);
         createNotificationList(this, resolvedNotificationPanel, resolvedNotificationListPanel,
-                resolvedNotificationLabel, resolvedNotifications, unresolvedNotificationList,
+                resolvedNotificationLabel, unresolvedNotificationList,
                 resolvedNotificationListScroller);
         this.frame.setSize(600, 600);
         this.frame.setVisible(true);
         this.frame.setTitle("Notifications");
         this.frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
     }
 
-    private void populateLists(String[] notifications) {
-        unresolvedNotifications = new String[]{"hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello",};
-        resolvedNotifications = new String[]{"hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello", "hello",};
+    public Notification[][] userNotifications(User user){
+        NotificationDatabaseInteractor notificationDBInteractor = new NotificationDatabaseInteractor();
+        ArrayList<Notification> notifications = notificationDBInteractor.readData();
+        ArrayList<Notification> unresolvedNotifications = new ArrayList<Notification>();
+        ArrayList<Notification> resolvedNotifications = new ArrayList<Notification>();
+        for (Notification n: notifications){
+            if (n.getResolvedStatus()){
+                if (n.getSenderId() == user.getEmployeeNum() || n.getRecipientId() == user.getEmployeeNum()){
+                    resolvedNotifications.add(n);
+                }
+            }
+            else{
+                if (n.getSenderId() == user.getEmployeeNum() || n.getRecipientId() == user.getEmployeeNum()){
+                    unresolvedNotifications.add(n);
+                }
+            }
+        }
+        Notification[][] noti = new Notification[][] {Notification.sortByCreatedDate(unresolvedNotifications), Notification.sortByCreatedDate(resolvedNotifications)};
```

Comment:
use case (not gui) class has to handle operations dealing directly with entities